r/R/type.R

```diff
@@ -56,7 +56,9 @@ DataType$import_from_c <- ImportType
 INTEGER_TYPES <- as.character(outer(c("uint", "int"), c(8, 16, 32, 64), paste0))
 FLOAT_TYPES <- c("float16", "float32", "float64", "halffloat", "float", "double")
 
-#' Infer the arrow Array type from an R object
+#' Infer Arrow type
+#'
+#' Infer the Arrow Array type from an R object.
```

Comment:
Could you say more about what this adds? Both lines say basically the same thing. What are we trying to convey with this?

Reply:
The L59 is treated as title in the Rd file.
The L61 is treated as description in the Rd file.
I thought that the title and description should be set separately, even if the content is similar.

I made the change in e7a375d95b5d3620ebdcef3c84111bcb9e3dfc67 because the roxygen2 tags can be used to clarify these intentions (this change does not affect the Rd file)

Reply:
> I thought that the title and description should be set separately, even if the content is similar.

Hmm, we have a [number](https://github.com/apache/arrow/blob/8b8914719cdfd6a18158783716562249025e9e63/r/R/schema.R#L335) [of other](https://github.com/apache/arrow/blob/8b8914719cdfd6a18158783716562249025e9e63/r/R/schema.R#L317) [places](https://github.com/apache/arrow/blob/8b8914719cdfd6a18158783716562249025e9e63/r/R/dictionary.R#L67)

Where we have the pattern of title-only roxygen blocks. If we don't have anything additional to say for the description in this block we should follow that same pattern and simply remove the "duplicated" line, IMO. Less code | fewer lines == less maintenance after all.

Reply:
I understand. I am sorry for the extra trouble I have caused you.